tools/windows/install-help/cal/FinalizeInstall.cpp

```diff
@@ -369,7 +369,7 @@ UINT doFinalizeInstall(CustomActionData &data)
         {
             DWORD lastErr = GetLastError();
             std::string lastErrStr = GetErrorMessageStr(lastErr);
-            WcaLog(LOGMSG_STANDARD, "CreateSymbolicLink: %s (%d)", lastErrStr.c_str(), lastErr);
+            WcaLog(LOGMSG_STANDARD, "CreateSymbolicLink: %S (%d)", lastErrStr.c_str(), lastErr);
```

Comment:
Either way.  I mean, you don't have to _guess_ if it's wide or not, but standardizing makes it easier.  I'll leave it to you.  I was just noting that the change went the wrong way.

Reply:
So, I would either use `std::wstring` or rename the function `GetErrorMessageStrW()` so that it's obvious what kind of string is being returned (I had to go look).